stable/jenkins/templates/jenkins-master-alerting-rules.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if and .Values.master.prometheus.enabled .Values.master.prometheus.alertingrules }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ template "jenkins.fullname" . }}
+  labels:
+    "app.kubernetes.io/name": '{{ template "jenkins.name" .}}'
+    "helm.sh/chart": "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    "app.kubernetes.io/managed-by": "{{ .Release.Service }}"
+    "app.kubernetes.io/instance": "{{ .Release.Name }}"
```

Comment:
I would add the label `"app.kubernetes.io/component": "{{ .Values.master.componentName }}"`

Reply:
@torstenwalter added, thanks